### PR TITLE
Update Office365ProPlus.nuspec

### PR DESCRIPTION
**Pull request by hanthor**
Adding a link to Microsoft's tool for creating a custom config.xml

### CODE REVIEW

File: manual/office365proplus/Office365ProPlus.nuspec [hunk 1]
```diff
@@ -13,14 +13,16 @@
...
+Example: `choco install office365proplus --params '/ConfigPath:c:\myConfig.xml` 
 
 * `/Language` - (optional) The installation language. Defaults to match your OS language. [List of possible languages](https://docs.microsoft.com/en-us/deployoffice/overview-of-deploying-languages-in-office-365-proplus#languages-culture-codes-and-companion-proofing-languages)
 Example: `choco install office365proplus --params '/Language:de-de'`
 
 Get the latest versions of your familiar Office programs, such as Word, Excel and PowerPoint, and more, all in one convenient
 subscription. This will download and silently install the entire Office 365 2016 Professional Plus edition. A license is required
 and you will be prompted to enter your account information upon first launching the software after install.
+      
+You can creat a custom config.xml at https://config.office.com/deploymentsettings
```
Comment (mikecole) [inline manual/office365proplus/Office365ProPlus.nuspec]: typo: 'creat'

**Comment by mikecole** [comment]
Thanks for the PR! I marked one typo. If you fix that, plus increment the second segment of the "version" to today's date, I'll go ahead and push. Thanks again!

**Comment by mikecole** [comment]
@hanthor  I just installed this package on a VM and did not see any references to 2019. Is there something I'm missing? I've always been a little confused about Office 365 product versioning. If you feel it should be 2019 let me know why.

**Comment by hanthor** [comment]
https://docs.microsoft.com/en-us/officeupdates/update-history-office365-proplus-by-date

**Comment by mikecole** [comment]
Good enough for me. If you want to adjust that in your PR I'll push it.
Thanks!

File: manual/office365proplus/Office365ProPlus.nuspec [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>office365proplus</id>
-    <version>2016.20190408</version>
+    <version>1903.20190408</version>
```
Comment (mikecole) [inline manual/office365proplus/Office365ProPlus.nuspec:6]: choco will not recognize this as a newer package.

**Comment by hanthor** [comment]
@mikecole shoot I’ll just change it back to 2019?

**Comment by mikecole** [comment]
I think that's probably best for now. I'm not sure why I changed the
versioning to 2016.* but it seems to have not been the right decision.

**Pull request closed without merging**